test/AutoDiff/autodiff_diagnostics.swift

```diff
@@ -157,22 +174,26 @@ enum T : P {
 }
 
 //===----------------------------------------------------------------------===//
-// Crasher TF-8
+// Classes and existentials (unsupported yet)
 //===----------------------------------------------------------------------===//
 
-protocol TF8Proto : Differentiable {
-  associatedtype Scalar
-  @differentiable(wrt: (self, input))
-  func applied(to input: Float) -> Float
+class Foo {
+  // @differentiable cannot be put here. It's rejected by Sema already.
+  func class_method(_ x: Float) -> Float {
+    return x
+  }
 }
 
-struct TF8Struct<Scalar> : TF8Proto where Scalar : FloatingPoint & Differentiable {
-  @noDerivative let bar: Scalar
-
-  @differentiable(wrt: (self, input))
-  // expected-note @+2 {{differentiating functions with parameters or result of unknown size is not supported yet}}
-  // expected-error @+1 {{function is not differentiable}}
-  func applied(to input: Float) -> Float {
-    return input
-  }
+// Nested call case.
+@differentiable // expected-error 2 {{function is not differentiable}}
```

Comment:
I'm not sure why this is emitted twice. My initial guess is it's because of the curry thunk, but I'll investigate it later.